models/nxos/logging_global/logging_global.yaml

```diff
@@ -0,0 +1,232 @@
+            type: str
+            choices: ["enabled", "disabled"]
+          level:
+            description: Set severity level for module logging.
+            type: int
+      monitor:
+        description: Set terminal line(monitor) logging level.
+        type: dict
+        suboptions:
+          state:
+            description: Enable or disable monitor logging.
+            type: str
+            choices: ["enabled", "disabled"]
+          level:
+            description: Set severity level for monitor logging.
+            type: int
+      origin_id:
+        description: Enable origin information for Remote Syslog Server.
+        type: dict
+        suboptions:
+          hostname:
+            description: Use hostname as origin-id of logging messages.
+            type: bool
+          ip:
+            description: Use ip address as origin-id of logging messages.
+            type: str
+          string:
+            description: Use text string as origin-id of logging messages.
+            type: str
+      rate_limit:
+        description: Enables rate limit for log messages.
+        type: bool
+      rfc_strict:
+        description: 
+          - Set RFC to which messages should compliant.
+          - Syslogs will be compliant to RFC 5424.
+        type: bool
+      servers:
+        description: Enable forwarding to Remote Syslog Servers.
+        type: list
+        elements: dict
+        suboptions:
+          state:
+            description: Enable or disable the syslog server.
+            type: str
+            choices: ["enabled", "disabled"]
+          facility:
+            description: Facility to use when forwarding to server.
+            type: str
+          port:
+            description: Destination Port when forwarding to remote server.
+            type: str
+          secure:
+            description: Enable secure connection to remote server.
+            type: dict
+            suboptions:
+              trustpoint:
+                description: Trustpoint configuration.
+                type: dict
+                suboptions:
+                  client_identity:
+                    description: 
+                      - Client Identity certificate for mutual authentication.
+                      - Trustpoint to use for client certificate authentication.
+                    type: str
+          use_vrf:
+            description: Display per-VRF information.
+            type: str
+      source_interface:
+        description: Enable Source-Interface for Remote Syslog Server.
+        type: str
+      timestamp:
+        description: Set logging timestamp granularity.
+        type: str
+        choices: ["microseconds", "milliseconds", "seconds"]          
+  state:
+    description:
+    - The state the configuration should be left in.
+    - Refer to examples for more details.
+    type: str
+    choices:
+    - merged
+    - replaced
+    - overridden
```

Comment:
Do we need overridden?

Reply:
@rohitthakur2590 Yes, overridden is a valid state.